pubspec.yaml

```diff
@@ -19,3 +23,7 @@ dev_dependencies:
   built_value_generator: '>=2.0.0 <5.0.0'
   source_gen: '^0.7.0'
   test: ^0.12.0
+
+transformers:
+  - test/pub_serve:
+      $include: test/**_test.dart
```

Comment:
[nit] AFAIK the `**` here isn't doing anything that `*` wouldn't.

Reply:
Recursive? I'll just keep it for now.

Reply:
I think recursive only applies when its is followed by a `/`